app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new; end
+
+  def create
+    @current_user = UsersController.current_user
+    comment_params = params.permit(:text)
+    @post = Post.find(params[:id])
+    @comment = @post.comments.new(text: comment_params[:text], author: @current_user)
+    puts "PARAMETERS:-------- #{params}"
```

Comment:
- [ ] Writing `puts` statement in the controller is not a good practice and is not recommended. Please remove that line.